src/test/java/com/google/sps/data/EventStorageTest.java

```diff
@@ -0,0 +1,171 @@
+        .setTitle("Party")
+        .setDescription("RSVP through the link")
+        .setCategory("Entertainment")
+        .setTags(LIST)
+        .setLocation("London, UK")
+        .setLocationId("ChIJdd4hrwug2EcRmSrV3Vo6llI")
+        .setDateTimeRange(RANGE_A)
+        .setDuration(new Long("30"))
+        .setLinks(LIST)
+        .setFields(MAP)
+        .setInvitedIDs(LIST)
+        .setJoinedIDs(LIST)
+        .setDeclinedIDs(LIST).build();
+
+  private static final Event EVENT_C = Event.newBuilder()
+        .setID("1")
+        .setGCalendarID("1")
+        .setOwnerID("1")
+        .setTitle("Graduation")
+        .setDescription("After the celebration join us for the party")
+        .setCategory("Entertainment")
+        .setTags(LIST)
+        .setLocation("Budapest, Hungary")
+        .setLocationId("ChIJyc_U0TTDQUcRYBEeDCnEAAQ")
+        .setDateTimeRange(RANGE_B)
+        .setDuration(new Long("30"))
+        .setLinks(LIST)
+        .setFields(MAP)
+        .setInvitedIDs(LIST)
+        .setJoinedIDs(LIST)
+        .setDeclinedIDs(LIST).build();
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  EventStorage eventStorageObject = new EventStorage();
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+    assertEquals(0, ds.prepare(new Query("Event")).countEntities(withLimit(10)));
+    ds.put(getEntityFromEvent(EVENT_A));
+    ds.put(getEntityFromEvent(EVENT_B));
+    ds.put(getEntityFromEvent(EVENT_C));
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  private Entity getEntityFromEvent(Event event) {
+    Entity entity = new Entity("Event");
+
+    entity.setProperty("gcalendar-id", event.getGCalendarID());
+    entity.setProperty("title", event.getTitle());
+    entity.setProperty("description", event.getDescription());
+    entity.setProperty("category", event.getCategory());
+    entity.setProperty("tags", event.getTags());
+    entity.setProperty("date", event.getDate());
+    entity.setProperty("time", event.getTime());
+    entity.setProperty("date-time-range", event.getDateTimeRangeAsJSON());
+    entity.setProperty("duration", event.getDuration());
+    entity.setProperty("location", event.getLocation());
+    entity.setProperty("location-id", event.getLocationId());
+    entity.setProperty("links", event.getLinks());
+    entity.setProperty("fields", event.getFieldsAsJSON());
+    entity.setProperty("owner", event.getOwnerID());
+    entity.setProperty("invited-users", event.getInvitedIDs());
+    entity.setProperty("joined-users", event.getJoinedIDs());
+    entity.setProperty("declined-users", event.getDeclinedIDs());
+
+    return entity;
+  }
+
+  @Test
+  public void eventStorageTest_getSearchedEvents_nullSearch_MatchesAll() {
+    assertEquals(3, eventStorageObject.getSearchedEvents(NULL_SEARCH).size());
```

Comment:
Maybe you should verify if the getSearchedEvents returns what it needs to return